### PR TITLE
Review get company file

### LINKED ISSUE #7: review get_company_file function
check we're happy with the way the function works, the syntax used, the code styling. Make any required changes, add guardian code if needed, checking function arguments for type and other expectations. Add/review roxygen headers

### PR DESCRIPTION
**Pull request by HarryJAlexander**
Reviewed and added roxygen headers to get_company_file.R file. Resolves #7

### CODE REVIEW

File: R/build_url.r [hunk 1]
```diff
@@ -27,3 +27,16 @@ build_url <- function(company_domain = .get_company_name(),
   url <- glue::glue("{base_url}/{company_domain}/{api_version}")
   return(url)
 }
+
```
Comment (tbowling87) [inline R/build_url.r:30]: we removed these 3 functions in one of the other PRs, so can drop these from this one as well

File: R/scratch.R [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# library(magrittr)
```
Comment (tbowling87) [inline R/scratch.R]: we deleted this file in one of the other PRs, so can delete it here as well

**Comment by HarryJAlexander** [comment]
The merge conflicts should be resolved now. Not sure if any further action is required on my end

File: R/get_company_file.R [hunk 3]
```diff
@@ -1,20 +1,38 @@
...
+#' "480",
+#' company_domain = "ascent",
+#' api_version = "v1",
+#' base_url = "https://api.bamboohr.com/api/gateway.php"
+#' )
+#'
+#'@author Mark Druffel, \email{mdruffel@propellerpdx.com}
+
+
+get_company_file <- function(file_id, company_domain, api_version, base_url){
```
Comment (tbowling87) [inline R/get_company_file.R]: can we give the last 3 arguments the same default values as build_url? That way the user can leave them out of the function call and will inherit from config file? So we want get_company_file("1") to work, as well as get_company_file("1", api_version = "v2) - if there ever is a v2

File: R/get_company_file.R [hunk 4]
```diff
@@ -1,20 +1,38 @@
...
 #'
-#' @description  fill in
+#' @description `get_company_file` takes a file_id (string), company domain (string),
+#' api version (string), and base_url (string) as arguments and then requests
+#' and returns data about the corresponding file from the BambooHR API.
 #'
-#' @param file_id  tbd
-#' @param ... tbd
+#' @param file_id The ID of the file to get from BambooHR.
+#' @param ... The company domain, API version, and base url.
```
Comment (tbowling87) [inline R/get_company_file.R]: give each of the three new params it's own entry

File: R/build_url.r [hunk 5]
```diff
@@ -27,3 +27,4 @@ build_url <- function(company_domain = .get_company_name(),
...
```
Comment (tbowling87) [inline R/build_url.r:30]: think this extra line may be spurious, without it, file should match that on develop branch

**Review by tbowling87 — APPROVED** [review]

**Pull request merged**